application/app/views/file_browser/_browser.html.erb

```diff
@@ -1,53 +1,74 @@
 <div class="card mb-3 overflow-hidden shadow-sm">
+  <h3 class="visually-hidden"><%= t('.header_file_browser_text') %></h3>
   <div class="card-header d-flex justify-content-between align-items-center">
     <!-- Path display (left) -->
-    <div data-file-browser-target="pathDisplay" class="d-flex align-items-center flex-wrap gap-2">
-      <% path_parts = current_path.split('/').reject(&:blank?) %>
-      <% accumulated_path = '/' %>
-      <% path_parts.each do |part| %>
-        <% accumulated_path = File.join(accumulated_path, part) %>
-        <span class="text-muted">/</span>
-        <span class="breadcrumb-folder"
-              data-entry-path="<%= accumulated_path %>"
-              data-entry-type="folder"
-              data-action="dblclick->file-browser#handleDoubleClick">
-          <i class="bi bi-folder-fill"></i> <%= part %>
-        </span>
-      <% end %>
-    </div>
+    <nav aria-label="<%= t('.nav_file_browser_label') %>">
+      <ol data-file-browser-target="pathDisplay" class="breadcrumb d-flex align-items-center flex-wrap gap-2">
+        <% path_parts = current_path.split('/').reject(&:blank?) %>
+        <% accumulated_path = '/' %>
+        <% path_parts.each do |part| %>
+          <li>
+            <% accumulated_path = File.join(accumulated_path, part) %>
+            <span class="text-muted">/</span>
+            <a class="breadcrumb-folder text-body text-decoration-none"
+               tabindex="0"
+               data-entry-path="<%= accumulated_path %>"
+               data-entry-type="folder"
+               data-action="dblclick->file-browser#handleDoubleClick keydown->file-browser#handleKeydown">
+              <i class="bi bi-folder-fill" aria-hidden="true"></i> <%= part %>
+            </a>
+          </li>
+        <% end %>
+      </ol>
+    </nav>
```

Comment:
path browser redefined as a ul list with li items inside a nav tag, to make it more understandable for accessibility devices. tabindex="0" and the `keydown` handler in `data-action` allow the usage of keyboard